hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bloom/HoodieBloomIndex.java

```diff
@@ -162,23 +162,22 @@ private List<Pair<String, BloomIndexFileInfo>> getBloomIndexFileInfoForPartition
    * Load all involved files as <Partition, filename> pair List.
    */
   List<Pair<String, BloomIndexFileInfo>> loadColumnRangesFromFiles(
-      List<String> partitions, final HoodieEngineContext context, final HoodieTable hoodieTable) {
```

Comment:
This code is a good example of when we were collecting intermediate results just to ask spark executors to do more work on each of these results.